grid_map_demos/src/move_demo_node.cpp

```diff
@@ -15,15 +12,16 @@ int main(int argc, char** argv)
   GridMap map({"layer"});
   map.setFrameId("map");
   map.setGeometry(Length(0.7, 0.7), 0.01, Position(0.0, 0.0));
-  ROS_INFO("Created map with size %f x %f m (%i x %i cells).\n The center of the map is located at (%f, %f) in the %s frame.",
+  ROS_INFO(
```

Comment:
Still ROS?

Reply:
This file is not ported into this PR, it is just linted.
This will be addressed in an upcoming PR.

Reply:
Thanks.